src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 17px;
+  padding-top: 33px;
+  padding-bottom: 18px;
```

Comment:
Avoid using multiple properties for padding if you can consolidate them into one. You can use `padding: 33px 17px 18px;` to combine `padding`, `padding-top`, and `padding-bottom`.